File: THCS/2022_2023/BinhPhuoc_22223/Bai1/tinhtong.py
# Đọc dữ liệu từ file tinhtong.inp
with open('tinhtong.inp', 'r') as infile:
    n = int(infile.readline().strip())

# Tính tổng các số chẵn nhỏ hơn n
if n > 2:
    k = (n // 2) - 1
    total = k * (n // 2)
else:
    total = 0

# Ghi kết quả vào file tinhtong.out
with open('tinhtong.out', 'w') as outfile:
    outfile.write(str(total) + '\n')

File: THCS/2022_2023/BinhPhuoc_22223/Bai1/tinhtong.cpp
#include <iostream>
#include <fstream>
using namespace std;

int main() {
    // Mở file để đọc dữ liệu
    ifstream infile("tinhtong.inp");
    ofstream outfile("tinhtong.out");

    long long n;
    infile >> n; // Đọc số nguyên dương chẵn từ file

    // Tính tổng các số chẵn nhỏ hơn n
    long long total = 0;
    if (n > 2) {
        long long k = (n / 2) - 1;
        total = k * (n / 2);
    }

    // Ghi kết quả vào file tinhtong.out
    outfile << total << endl;

    // Đóng file
    infile.close();
    outfile.close();

    return 0;
}
